### DISCUSSION TITLE
I have a student ID but the VALID UNTIL DATE is in the back of my ID. It's hard to apply for the GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Crysil25** [community/community#60662] [GitHub Education]
### Select Topic Area

General

### Body

Anyone can help me? I know if you apply for the GitHub Student Developer Pack you need an STUDENT ID with VALID UNTIL DATE. But my problem is my VALID UNTIL DATE is in the BACK. Every time I send my ID it is automatically REJECTED because there is no VALID UNTIL DATE of my front ID. I hope you will help me about this. Thanks

**Comment by kyoto7250** [comment]
I also met the exact same problem. 

My ID card also had an expiration date on the back, but my name was on the front, so when  I tried to upload the expiration date of the ID card, but it was rejected because my name did not exist in the photo. This was a very stressful experience.

I tried to show the front and back of my ID card, but since uploading multiple images is not supported, I was unable to send the expiry date.

In the end, I only sent the front side. I would appreciate it if you could consider a pattern that has the name and expiration date on both sides of the ID card.

**Comment by Dineth-H** [comment]
Hey there!...I also faced the same issue. My student ID card has the expiration date on the back, and when I try to upload a photo of it, my application is rejected because my name in the not in the photo as it is on the front.

I've tried uploading a photo of both front and back of my ID, but GitHub doesn't let me upload multiple images.

so what I did was I took the front and back and merged to one jpeg and tried uploading. Hooray! I already received the the student benefits and are currently enjoying. hope the same works for you!

**Reply by chanu0073** [reply]
But now in 2025, there is no option to upload image. There's only one option i.e, taking a picture with the camera, we cannot upload any file. What should I do now, any suggestions?

**Reply by FarazHayder** [reply]
Click on upload image and it will automatically display the option for "Upload a file"

**Reply by chanu0073** [reply]
There's no "Upload a file" option in 2025 now.
![Screenshot 2025-07-05 200704](https://github.com/user-attachments/assets/7fcb24b7-bfd2-4ad9-8186-b1090de595aa)

**Reply by balintuszi1** [reply]
I have the same issue, there is no longer an option to upload a file

**Reply by giri3105** [reply]
same here  -_-

**Reply by balintuszi1** [reply]
> same here  -_-

I ended up scanning my ID and printing both sides on a paper, this way I could upload it as one image and it got accepted in a few minutes.

**Comment by DrakeYounger** [comment]
I remember when I was in your shoes, trying to apply for all these awesome deals with my student ID, only to realize the expiration date was playing hide and seek on the back. It's frustrating, to say the least!

Have you tried reaching out to GitHub's support team? Sometimes they can be pretty understanding and might be able to help you out, given your unique situation.

Oh, and speaking of IDs, I came across one source while browsing about where I can get a fake ID - you might want to check out this link: [https://fakeyourdrank.com/how-to-pay-for-a-fake-id](https://fakeyourdrank.com/how-to-pay-for-a-fake-id). Just in case you need a backup plan!

**Comment by kacper516** [comment]
I also had the same issue, and as mentioned above, I merged two images, the front and back images, and combined them into one using Paint.

**Comment by tattoosonmyskin** [comment]
im attending an online school and my stdent id doesnt contain any date other than my expected graduation date. what other information do i need? they accepted my student email and confirmed it, what are my options

**Reply by balintuszi1** [reply]
Does it have some sort of a sticker on it? (I need to renew to sticker on mine every year and that's how it gets valid.) I would still try it like that, if it gets denied then ask the online school about the date of validness, or ask them how you can apply for student discounts.

If it is still flagged as invalid by github, you can upload a proof of enrolment and it should work too.
